mos-platform/fds/fds_bios_LoadFiles.s

```diff
@@ -0,0 +1,30 @@
+.text
+
+__fds_bios_LoadFiles:
+  jsr __LOAD_FILES
+  .word $ADDE
+  .word $EFBE
+  rts
+
+;
+; char fds_bios_LoadFiles (struct fds_bios_disk_id *disk_id, char *file_ids, char *loaded);
+;
+.global fds_bios_LoadFiles
+fds_bios_LoadFiles:
+  lda __rc2
+  sta [__fds_bios_LoadFiles+3]
+  lda __rc3
+  sta [__fds_bios_LoadFiles+4]
+  lda __rc4
+  sta [__fds_bios_LoadFiles+5]
+  lda __rc5
+  sta [__fds_bios_LoadFiles+6]
+  jsr __fds_bios_LoadFiles
```

Comment:
I think these wrappers could be implemented both more clearly and more efficiently as follows:

```
fds_bios_LoadFiles:
  lda __rc2
  sta .Larg1
  lda __rc3
  sta .Larg1+1
  lda __rc4
  sta .Larg2
  <...>
  jsr __LOAD_FILES
.Larg1:
  .word $ADDE
.Larg2:
  .word $EFBE
  bne
  <...>
```

Am I missing a reason for `__fds_bios_Loadfiles` to be broken out into its own helper? In any case, using local symbols to name the arguments should always make code like this clearer.